pkg/release/dependency/graph_test.go

```diff
@@ -120,6 +120,86 @@ func (s *GraphTestSuite) TestCycles() {
 	s.Require().Error(graph.Build())
 }
 
+// 1 -> 3 -> 4
+// 2 /
+// Reversed
+// 4 -> 3 -> 1
+// --------> 2
+func (s *GraphTestSuite) TestReverse() {
+	graph := dependency.NewGraph[string, string]()
+
+	s.Require().NoError(graph.NewNode("1", "1"))
+	s.Require().NoError(graph.NewNode("2", "2"))
+	s.Require().NoError(graph.NewNode("3", "3"))
+	s.Require().NoError(graph.NewNode("4", "4"))
+
+	graph.AddDependency("3", "1")
+	graph.AddDependency("3", "2")
+	graph.AddDependency("4", "3")
+
+	s.Require().NoError(graph.Build())
+	graph, err := graph.Reverse()
+	s.Require().Nil(err)
```

Comment:
`NoError` would be nicer just because it's an error, functionally it's the same

Reply:
done